interfaces/Portalicious/package.json

```diff
@@ -17,8 +17,9 @@
     "lint": "ng lint",
     "fix": "ng lint --fix",
     "test:all": "npm run lint && npm test",
-    "test:ci": "ng test --no-watch --no-progress --browsers=ChromeHeadless",
-    "test": "ng test",
+    "test:deployment-config": "node ./_test-deployment-config.js",
+    "test:ci": "npm run test:deployment-config && ng test --no-watch --no-progress --browsers=ChromeHeadless",
+    "test": "npm run test:deployment-config && ng test",
```

Comment:
Not a fan of these two changes. I would rather add an extra line in the github test yaml and _only_ test it there, rather than run this test every time I want to run a unit test.